x-pack/plugins/siem/cypress/screens/hosts/events.ts

```diff
@@ -6,6 +6,9 @@
 
 export const CLOSE_MODAL = '[data-test-subj="modal-inspect-close"]';
 
+export const DRAGGABLE_HEADER =
```

Comment:
I am wondering if it is best to move those declarations to `timeline.ts` as it is the same for all timeline pages. I had this dilemma for myself when I started writing the cypress test for my [PR](https://github.com/elastic/kibana/pull/67156). What do you think is best to do?

Reply:
I agree @cnasikas , and moved the definition of `DRAGGABLE_HEADER` from `events.ts` to `timeline.ts` as suggested, because this constant is also applicable to timelines too (not just the Events view).

With that same commit, I also added a `.skip` to the `Events columns` test for now, because although it passed CI on Friday, I ran `pack/plugins/siem/scripts/loop_cypress_tests.js` over the weekend and observed a few failures. Other Cypress tests that are currently passing CI are also failing locally, suggesting the failure _may_ be a local environment issue, but I don't want to risk destabilizing CI, and this PR must be backported soon to `7.8` for the next BC.

@MadameSheema, if you're willing, perhaps we can pair on re-enabling it in a separate PR?